### ISSUE TITLE
Limit for number of images and additional price

### ISSUE THREAD

**Issue by boobiq** [allegro/allegro-api#295]
Hi,

I'd like to know how many images can be added to the offer and if there is additional price for them.

When I try to add offer manually, it says "Możesz dodać 10 zdjęć" and total price for listing the offer doesn't change depending on how many images I add.

But when I get fids from **DoGetSellFormFieldsForCategory**, there are only 8 fids for images with description:

```
...
Możesz dodać 8 zdjęć. Pierwsze zdjęcie do 50 kB gratis, każde kolejne - 0,10 zł.
Maksymalny rozmiar zdjęcia nie może przekroczyć 200 kB. Za zdjęcia powyżej 50 kB - dopłata 0,10 zł.
...
```

but I always get the same price back in response after listing a new offer with **DoNewAuctionExtRequest** regardless of number or dimensions of images.

I suppose I can only upload up to 8 images through WebAPI since there is no more fids for them, but will it cost more for 2nd to 8th images and for larger images? Or is it just some old description and price doesn't change with images anymore?

Thanks for help!

**Comment by KoniecWebAPI-KoniecAllegro** [comment]
Wow !
The Doc's (still) say: up to 8 PICs ;-)

**Comment by PawelTaberski** [comment]
@boobiq In DoGetSellFormFieldsForCategor request what category do you give?

**Comment by boobiq** [comment]
@PawelTaberski I tried it again and searched for "Zdjęcie" and just now I noticed there are other fids for Zdjęcie 9 - 16 which I didn't see before since it's almost at the end of response and I thought it ends at Zdjęcie 8.

My mistake, I'm sorry.

So my questions now are:
- can I use fids for image 9-16 only and skip 1-8?
- is it free regardless of number of images? (I mean number of images doesn't affect price for listing an offer)
- technically it's possible to use all fids to have 16 images?

Thanks
